### ISSUE TITLE
-warp Z

### ISSUE THREAD

**Issue by Sooleawa2** [zajako/LineageHD#438]
I noticed that you can stand by the locked doors at -warp z and mobs will come to you and attack through the door. Sometimes they come through the door. You can hit them with an area while standing at the door.

**Comment by zajako** [comment]
Yes I’m aware of the walk through door bug for monsters and briefly tried to fix it before release but ran out of time to do so. I’ll continue to look into it when I get a chance
